kalite/version.py

```diff
@@ -7,7 +7,7 @@
 # Must also be of the form N.N.N for internal use, where N is a non-negative integer
 MAJOR_VERSION = "0"
 MINOR_VERSION = "14"
-PATCH_VERSION = "0"
+PATCH_VERSION = "alpha7"
```

Comment:
Don't use version numbers like this. distutils will normalize the version number to something else (with e.g. `python setup.py sdist --static`) which causes problems when other components get the version number from `version.py` and expect something else. Please just user numbers, as the comment above suggests.